gap/examples.gd

```diff
@@ -48,3 +48,7 @@ DeclareOperation("PetersenGraph", [IsFunction]);
 DeclareConstructor("GeneralisedPetersenGraphCons", [IsDigraph, IsInt, IsInt]);
 DeclareOperation("GeneralisedPetersenGraph", [IsInt, IsInt]);
 DeclareOperation("GeneralisedPetersenGraph", [IsFunction, IsInt, IsInt]);
+
+DeclareConstructor("TadpoleDigraphCons", [IsDigraph, IsInt, IsInt]);
+DeclareOperation("TadpoleDigraph", [IsInt, IsInt]);
+DeclareOperation("TadpoleDigraph", [IsFunction, IsInt, IsInt]);
```

Comment:
If you're happy to disallow the second argument to be `0`, then please make the following change, and similarly please update the method installations to be `IsPosInt` rather than `IsInt` (and correspondingly, the error message for `n < 0` and any code relating to `n = 0` can go).
```suggestion
DeclareConstructor("TadpoleDigraphCons", [IsDigraph, IsPosInt, IsPosInt]);
DeclareOperation("TadpoleDigraph", [IsPosInt, IsPosInt]);
DeclareOperation("TadpoleDigraph", [IsFunction, IsPosInt, IsPosInt]);
```